### PR TITLE
Arjan development

### PR DESCRIPTION
**Pull request by arjan-bal**
Made portfolio front-end. All changed files located in portfolio\src\main\webapp. Open index.html to view. Also deployed at https://arjan-bal.github.io
Used Material Design Bootstrap and used components from other's projects

### CODE REVIEW

**Comment by abhatia2107** [comment]
Will suggest you to create a separate PR for all the server-side changes.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,524 @@
...
+          </div>
+      </section>
+    </div>
+
+    <footer class="page-footer text-center font-medium mt-4 wow fadeIn">
+      <!--Footer-->    
+      <!--Copyright-->
+      <div class="footer-copyright py-4">
+          © 2020 Copyright | 
+          <a href="https://github.com/sonumeewa" target="_blank">Arjan <strong>Singh Bal</strong> </a>
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:506]: sonumeewa?

File: portfolio/src/main/webapp/js/script.js [hunk 2]
```diff
@@ -26,3 +37,22 @@ function addRandomGreeting() {
...
+    );
+  }
+}
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/js/script.js:58]: newline

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,524 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body onload="getAnnouncements()">
+
+    <nav class="navbar fixed-top navbar-expand-lg navbar-dark scrolling-navbar top-nav-collapse">
+      
+      <div class="container">
+
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:25]: nit: Be consistent with newlines, it makes code more readable.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,524 @@
...
+          <div class="col-12 col-sm-5 align-middle">
+            <img class="pull-right avatar avatar-lg ml-4 rounded-circle " src="img/arjan.jfif" alt="" >
+          </div>
+
+          <div class="col-12 col-sm-7 text-left pl-5">
+            <h1 class="display-4 mb-2 mt-2 h1">
+              <strong>Arjan Singh Bal</strong>
+            </h1>
+
+            <p class="mb-0 p" style="font-size: 1.250rem; font-weight: 200;">
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:105]: Never use inline css/
https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css

Comment (arjan-bal) [reply portfolio/src/main/webapp/index.html]: Got it, removed inline styles

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (arjan-bal) [reply portfolio/src/main/webapp/index.html]: Made changes

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (arjan-bal) [reply portfolio/src/main/webapp/index.html]: He's a college friend, I've used the footer from one of his projects. Forgot to remove the hyperlink 😅

File: portfolio/src/main/webapp/css/style.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  .pull-right {
+    float: right;
+  }
+  .carousel-indicators, .carousel-control-prev, .carousel-control-next {
+    z-index: 30;
+  }
+
+  .pr-1 {
+      color: white;
+  }
```
[the remark this answers is not in the document]
Comment (arjan-bal) [reply portfolio/src/main/webapp/css/style.css]: Learnt something new. Added newlines to all files

File: portfolio/src/main/webapp/css/style.css [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  
+    .carousel-content {
+      top: 20%!important;
+    }
+  
+    .carousel-content div {
+      font-size: 1rem!important;
+    }
+
```
[the remark this answers is not in the document]
Comment (arjan-bal) [reply portfolio/src/main/webapp/css/style.css]: Changed

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 7]
```diff
@@ -14,6 +14,8 @@
```
[the remark this answers is not in the document]
Comment (arjan-bal) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:14]: Okay

File: portfolio/src/main/webapp/js/script.js (hunk 2 above)
Comment (arjan-bal) [reply portfolio/src/main/webapp/js/script.js:58]: Added newline

File: portfolio/src/main/webapp/css/style.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  
+    .carousel-content div {
+      font-size: 1rem!important;
+    }
+  
+  }
+  
+  @media (max-height: 576px) and (orientation : landscape){
+
```
[the remark this answers is not in the document]
Comment (arjan-bal) [reply portfolio/src/main/webapp/css/style.css]: Made changes throughout

**Pull request merged**